guava/src/com/google/common/escape/ArrayBasedCharEscaper.java

```diff
@@ -101,6 +102,7 @@ protected ArrayBasedCharEscaper(ArrayBasedEscaperMap escaperMap, char safeMin, c
    * This is overridden to improve performance. Rough benchmarking shows that this almost doubles
    * the speed when processing strings that do not require any escaping.
    */
+  @SuppressWarnings("array.access.unsafe.low")//char types are non negative: https://github.com/kelloggm/checker-framework/issues/192
```

Comment:
Here and elsewhere, please prefix each warning suppression with the name of the checker which is issuing the warning. So, for instance, here you should write `lowerbound:array.access.unsafe.low`. Though it's not that important in this case (only the LBC will ever issue `array.access.unsafe.low` errors...), it's important when suppressing a more general warning so that the application is not too broad.